packages/DropdownMenu/src/components/Item/Item.js

```diff
@@ -28,27 +31,39 @@ const propTypes = {
 const defaultProps = {
   isDestructive: false,
   onClick: () => {},
+  onKeyDown: () => {},
   onClose: () => {},
   onShowConfirmation: () => {},
   renderConfirmation: null,
 };
 
 const Item = props => {
-  const { children, isDestructive, onClick, onClose, onShowConfirmation, renderConfirmation, ...moreProps } = props;
+  const {
+    children,
+    isDestructive,
+    onClick,
+    onClose,
+    onShowConfirmation,
+    renderConfirmation,
+    onKeyDown,
+    ...moreProps
+  } = props;
 
   const handleClickItem = () => {
-    onClick();
+    onClick(children);
```

Comment:
returning the children here to the consuming application click handler, although perhaps it isn't required as they can get the value themselves as the onClick is on each item.. 🤷‍♂ I decided it doesn't hurt to pass it though